spec/models/export/submissions_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+require 'stringio'
+
+RSpec.describe Export::Submissions do
+  context 'given valid submissions and an in-memory output' do
```

Comment:
is "valid" the right word here? one of the "valid" submissions in this case has failed validation.

Reply:
It's a little overloaded. I'll go with 'some' :smile: